packages/redux/src/sizePreferences/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,165 @@
+import * as actionTypes from '../actionTypes.js';
+import { type ListingFacet, toBlackoutError } from '@farfetch/blackout-client';
+import {
+  mockListingFacets,
+  mockQuery,
+} from 'tests/__fixtures__/sizePreferences/index.mjs';
+import reducer, * as fromReducer from '../reducer.js';
+import type { SizePreferencesState } from '../types/state.types.js';
+
+let initialState: SizePreferencesState;
+const randomAction = { type: 'this_is_a_random_action' };
+
+describe('sizePreferences reducer', () => {
+  beforeEach(() => {
+    initialState = fromReducer.INITIAL_STATE;
+  });
+
+  describe('reset handling', () => {
+    it('should return the initial state', () => {
+      expect(
+        reducer(undefined, {
```

Comment:
Use a different value instead of undefined for the state so that the action is properly tested.